### PR TITLE
Edit Content Standards

### PR DESCRIPTION
**Pull request by yangc95**
### Description

To avoid any confusion with file naming for terms that are in camelCase or PascalCase

### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: documentation/content-standards.md [hunk 1]
```diff
@@ -47,7 +47,9 @@ We'll describe the standards for each of these components separately.
 All entry file names use the `.md` format along with the following standards:
 
 - They must match the title of the entry. (e.g., "Three.js" -> `three-js.md`)
-- It must be separated by dashes, where appropriate. (e.g., "Data Types" -> `data-types.md`, "HashTable" -> `hashtable.md`)
+- It must be separated by dashes, where appropriate. (e.g., "Data Types" -> `data-types.md`)
+  - If the title or name is meant to be in PascalCase: "HashTable" -> `hashtable.md`
+  - If the title or name is meant to be in camelCase: ".forEach()" -> `forEach.md`.
```
Comment (Dusch4593) [inline documentation/content-standards.md:52]:
```suggestion
  - If the title or name is meant to be in camelCase: ".forEach()" -> `forEach.md` (without the `.` and `()`).
```

File: documentation/content-standards.md [hunk 2]
```diff
@@ -47,7 +47,9 @@ We'll describe the standards for each of these components separately.
...
 - The name should always be lowercase. (e.g., "Array" -> `array.md`, "HTML" -> `html.md`)
 - Punctuation must not be included in the file name, even if it exists in the entry name. (e.g., "Moore's Law" -> `moores-law.md`, "Three.js" -> `three-js.md`)
```
Comment (Dusch4593) [inline documentation/content-standards.md:54]:
```suggestion
- Punctuation must not be included in the file name, even if it exists in the entry name. (e.g., "Moore's Law" -> `moores-law.md`, "Three.js" -> `three-js.md`, `.nextAfter()` -> `nextAfter.md`)
```

File: documentation/content-standards.md [hunk 3]
```diff
@@ -47,7 +47,9 @@ We'll describe the standards for each of these components separately.
 All entry file names use the `.md` format along with the following standards:
 
 - They must match the title of the entry. (e.g., "Three.js" -> `three-js.md`)
-- It must be separated by dashes, where appropriate. (e.g., "Data Types" -> `data-types.md`, "HashTable" -> `hashtable.md`)
+- It must be separated by dashes, where appropriate. (e.g., "Data Types" -> `data-types.md`)
+  - If the title or name is meant to be in PascalCase: "HashTable" -> `hashtable.md`
+  - If the title or name is meant to be in camelCase: ".forEach()" -> `forEach.md`.
 - The name should always be lowercase. (e.g., "Array" -> `array.md`, "HTML" -> `html.md`)
```
Comment (Dusch4593) [inline documentation/content-standards.md:53]: As its currently phrased, I'm not sure how this squares up with the list items above. Maybe we could rephrase as something like this:

```suggestion
- The name should always begin with a lowercase letter. (e.g., "Array" -> `array.md`, "HTML" -> `html.md`)
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@yangc95 Had a few comments/suggestions. Thank you for opening this! 😎

**Review by Dusch4593 — APPROVED** [review]
@yangc95 LGTM! 👍🏻

Anything left to add, change, or check @lucwhite?

**Review by KTom101 — APPROVED** [review]

**Review by HishamT — APPROVED** [review]
LGTM

**Pull request merged**